docs_src/examples/Ch-OSImageWithEdgeX.md

```diff
@@ -373,193 +279,107 @@ logout
 Connection to localhost closed.
 ```
 
-... and query data from outside
+... and query data from outside via the API Gateway:
 ```bash title="🖥 Desktop"
-curl --silent --show-err http://localhost:59880/api/v2/reading/all?limit=2 | jq
+curl --insecure https://localhost:8443/core-data/api/v3/reading/all?limit=2
 ```
 
-```json title="Response"
-{
-  "apiVersion": "v2",
-  "statusCode": 200,
-  "totalCount": 7040,
-  "readings": [
-    {
-      "id": "5907adb3-18f6-4cf9-9826-216285b7f519",
-      "origin": 1660225195484643300,
-      "deviceName": "Random-Integer-Device",
-      "resourceName": "Int32",
-      "profileName": "Random-Integer-Device",
-      "valueType": "Int32",
-      "value": "445276022"
-    },
-    {
-      "id": "86c87583-a220-44d9-b29a-e412efa97cc3",
-      "origin": 1660225195423943200,
-      "deviceName": "Random-Integer-Device",
-      "resourceName": "Int64",
-      "profileName": "Random-Integer-Device",
-      "valueType": "Int64",
-      "value": "-6696458089943342374"
-    }
-  ]
-}
-```
-
-However, as expected, we can't access securely via the API Gateway:
-```bash title="🖥 Desktop"
-curl --insecure https://localhost:8443/core-data/api/v2/reading/all?limit=2
-```
-
-```json title="Response"
-{"message":"Unauthorized"}
-```
+Since the security is enabled, the access is not authorized.
+You can follow the instructions from the [getting started](../../getting-started/Ch-GettingStartedSnapUsers/#adding-api-gateway-users) to add a user to API Gateway, and generate a JWT token to access the API securely.
 
-You can follow the instructions from the [getting started](../../getting-started/Ch-GettingStartedSnapUsers/#adding-api-gateway-users) to create a key pair, add a user to API Gateway, and generate a JWT token to access the API securely.
+---
+{==
```

Comment:
It is a good idea to highlight each chapter's summary! The == syntax indicates highlighting, which is rendered as a very bright yellow, resembling a warning. Is it possible to change the color to something like green?

Reply:
Unfortunately, highlighting with a different color is not possible: https://squidfunk.github.io/mkdocs-material/reference/formatting/

But this is the typical color to highlight text. Warnings are more orange.